index.js

```diff
@@ -760,23 +766,32 @@ var tick = function() {
 				if(!("units" in node)) {
 					node.units = {};
 				}
-				if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
-					if(node.owner != -1) {
-						nodeWinner = node.owner;
+				if (node.owner == -3) {
+					for (var owner in node.units) {
+						// convert all units to energy
+						gd.data.energy[owner] = Math.min(gd.data.energy[owner] + node.units[owner], GAMERULES.MAX_ENERGY);
+						node.units[owner] = 0;
+						logins[gd.users[adjustForRemoved(gd, owner)]].conn.send("energy:" + gd.data.energy[owner]);
 					}
-				}
-				else {
-					debugMsg(gd, "winning", "contested due to node owner");
-					debugMsg(gd, "winning", nodeWinner, node.owner, typeof nodeWinner, typeof node.owner);
-					unitsUncontested = false;
-				}
-				for(var owner in node.units) {
-					if(owner != node.owner && node.units[owner] > 0) {
-						debugMsg(gd, "winning", "contested due to units");
+				} else {
+					if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
+						if(node.owner != -1) {
+							nodeWinner = node.owner;
+						}
+					}
+					else {
+						debugMsg(gd, "winning", "contested due to node owner");
+						debugMsg(gd, "winning", nodeWinner, node.owner, typeof nodeWinner, typeof node.owner);
 						unitsUncontested = false;
 					}
+					for(var unitOwner in node.units) {
+						if(unitOwner != node.owner && node.units[unitOwner] > 0) {
+							debugMsg(gd, "winning", "contested due to units");
+							unitsUncontested = false;
+						}
+					}
 				}
-				if(node.owner != -1) {
+				if(node.owner != -1 && node.owner != -3) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2675148799/issues/source?bid=3248399&fileBranchId=3307983#l794)